Formula/pdal.rb

```diff
@@ -12,9 +11,14 @@ class Pdal < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "boost"
   depends_on "gdal"
   depends_on "laszip" => :optional
+  depends_on "postgresql" => :optional
+  depends_on "curl" => :optional
+  depends_on "python" => :optional
+  depends_on "pcl" => :optional
+  depends_on "hdf5" => :optional
+  depends_on "sqlite" => :optional
```

Comment:
`sqlite` can be made non-optional, because it's already a dependency of `gdal`

Reply:
will do.